packages/docs/community/release-process/attestation-service.md

```diff
@@ -0,0 +1,143 @@
+# Attestation Service Release Process
+
+## Versioning
+
+Releases of Attestation Service are numbered according to semantic versioning, as described at [semver.org](https://semver.org).
+
+New releases can be expected as follows:
+
+* Major releases: approximately yearly
```

Comment:
Not really. But without historical data, major releases being released approximately yearly is fine to say imo. It's more to set an expectation that major releases don't happen all the time. I believe this is the timeline on most of the release docs.

Reply:
yeah, it's tough. We went 6 months without a release then have made 4 point releases and 2 minor in two months. But I don't expect we'll keep that cadence. I'll cut since I don't feel the prediction is useful